src/compiler/scala/reflect/quasiquotes/Reifiers.scala

```diff
@@ -130,7 +130,7 @@ trait Reifiers { self: Quasiquotes =>
       case Placeholder(Hole(tree, NoDot)) if isReifyingPatterns => tree
       case Placeholder(hole @ Hole(_, rank @ Dot())) => c.abort(hole.pos, s"Can't $action with $rank here")
       case TuplePlaceholder(args) => reifyTuple(args)
-      // Due to greediness of syntactic applied we need to pre-emptively peek inside.
+      // Due to greediness of syntactic applied we need to preemptively peek inside.
```

Comment:
This is also UK vs US.

* https://en.oxforddictionaries.com/definition/pre-emptively
* https://en.oxforddictionaries.com/definition/us/preemptively

(Btw, I'm mostly mentioning it for its trivia value)